force-app/src/classes/Mock.cls

```diff
@@ -9,9 +9,6 @@ public class Mock implements System.StubProvider {
   public Object stub { get; private set; }
   private Map<String, MethodSpy> spies = new Map<String, MethodSpy>();
 
-  private Mock() {
```

Comment:
question: is there any possibility that the default empty constructor could be `public` by default ? I'd like to be sure that removing this constructor does not open the API to build `Mock` with `new Mock()`

Reply:
@LudoMeurillon it seems the only entry point with the other constructor being private is through the static factory method `Mock.forType`, so I don't see how this could affect anything else. I had the same thought prior to seeing that.

Reply:
I just tried it locally and it is not possible to instanciate a Mock even if the default constructor is not private